CMakeLists.txt

```diff
@@ -80,13 +80,10 @@ set(SOURCES
     src/spawner_pcms2.cpp
 )
 
-set(LIBRARIES libspawner)
+set(LIBRARIES "libspawner")
```

Comment:
It seems most of existing code does not use quotes.
I would prefer this way unless there is compelling reason to quote.